src/web/routes/cookies.js

```diff
@@ -1,10 +1,13 @@
-const pageContent = {
+const { getLanguageBase } = require('./application/common/language')
+
+const pageContent = ({ language }) => ({
   // TODO setup cookies page to handle translations HTBHF-586
```

Comment:
Do we need to add a cy cookie page?

Reply:
Yep, that's the next sub-task. There's some work around getting the Cookie link name and translations so thought I'd keep that all together.